docs/BreakingChanges/037 New enum values in WPF's PageRangeSelection.md

```diff
@@ -0,0 +1,27 @@
+## 37: New enum values in WPF's PageRangeSelection
+
+### Scope
+Edge
+
+### Version Introduced
+4.5
+
+### Change Description
+Two new members (CurrentPage and SelectedPage) have been added to the <a href="https://msdn.microsoft.com/en-us/library/system.windows.controls.pagerangeselection(v=vs.110).aspx">PageRangeSelection</a> enum.
+
+- [ ] Quirked
+- [ ] Build-time break
+- [ ] Source analyzer available
+
+### Recommended Action
+In most cases, these changes won't impact user code. Code that depends on a particular number of elements existing in Enum.GetNames or Enum.GetValues calls on the PageRangeSelection type should be modified, though.
+
+### Affected APIs
+* `T:System.Windows.Controls.PageRangeSelection`
+
+[More information](https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#wpf)
+
+<!--
+    ### Notes
+    Look for cast to PageRangeSelection, or Enum.GetNames or Enum.GetValues calls taking this type as input
```

Comment:
Do you want a 'source analyzer status' in the notes here since no analyzer is available yet? Or are you not tracking analyzers with that in these files?